docs/Sonarr/sonarr-collection-of-custom-formats.md

```diff
@@ -1146,6 +1146,46 @@ We've made 3 guides related to this.
 
 ---
 
+### Single Episode
+
+!!! danger "WARNING"
+
+    - This CF utilizes functionality only available starting in Sonarr v4.0.2.1262.
+
+??? question "Single Episode - [Click to show/hide]"
+
+    This Custom Format will recognize single episode releases (as opposed to multi-episode or season packs) using Sonarr's built in parser. Depending on your preferences, you can give it a negative score, a positive score, or just skip adding it altogether.
```

Comment:
```suggestion
        This Custom Format will recognize single-episode releases, instead of multi-episode or season packs, using Sonarr's built-in parser. Depending on your preference, you can give it a negative or positive score or skip adding it altogether.